src/components/Layout/LanguageSelector.tsx

```diff
@@ -0,0 +1,131 @@
+import Badge from '@ably/ui/core/Badge';
+import Icon from '@ably/ui/core/Icon';
+import { IconName } from '@ably/ui/core/Icon/types';
+import cn from '@ably/ui/core/utils/cn';
+import { useRef, useState } from 'react';
+import Select from 'react-select';
+import { languageData } from 'src/data/languages';
```

Comment:
we might able to save 1 line by combining the two imports?

`import { languageData, languageInfo } from 'src/data/languages';`